package.json

```diff
@@ -2,9 +2,14 @@
   "name": "jsonc-parser",
   "version": "3.2.1",
   "description": "Scanner and parser for JSON with comments.",
-  "main": "./lib/umd/main.js",
-  "typings": "./lib/umd/main.d.ts",
-  "module": "./lib/esm/main.js",
+  "main": "./lib/cjs/main.js",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/main.js",
+      "module": "./lib/esm/main.js",
```

Comment:
What's the meaning of `'module'` exports? I didn't see it in nodejs docs and webpack docs. Should we change it to `"require": "./lib/cjs/main.js"`?

Reply:
It tells bundlers to use it, even when the module is `require`d from a CJS file. So it helps reduce bundle size and avoid the dual package hazard.